### DISCUSSION TITLE
my apologies

### DISCUSSION THREAD

**Discussion by aliparlakci** [Serene-Arc/bulk-downloader-for-reddit#767] [General]
Hi everyone!

I hope everyone is doing great.

I have been neglecting bdfr for a very long time. I was busy with my school, work and some personal matters. I am so sorry if it caused you any trouble. I will actively participate in the discussions and, hopefully, the development of the project, from now on.

It is awesome to see that new contributors are joining to the project and it feels great to have started something that is built collectively. I am amazed by the work you did during my absence. I have never thought this project would grow this big.

I think it is time for a new official release after 2 years. Let me know what do you think.

Great job!
Take care,
Ali

**Comment by aliparlakci** [comment]
@Serene-Arc @OMEGARAZER @Soulsuck24

**Comment by OMEGARAZER** [comment]
Are there any goals or features you're looking for in a 2.7 release? 2.6.2 just went out the other day so doesn't seem like there's a need to rush it out or anything.

I've currently got a gfycat API change that should be done shortly and slowly going through and linting/refactoring stuff.

**Comment by aliparlakci** [comment]
Oh, I thought the packages on PyPI were stale since my last version update. Sorry, my mistake. Then this is not necessary.
